src/datasources.js

```diff
@@ -357,7 +362,7 @@ define([
             _.each(nodes, function (node) {
                 if (node.hashtag && !node.index) {
                     hashtags.map[node.hashtag] = node.xpath;
-                    hashtags.transforms[node.hashtagPrefix] = function (prop) {
+                    hashtags.transforms[node.sourceInfo.hashtag + '/'] = function (prop) {
```

Comment:
Do you mean an assertion? `node.sourceInfo.hashtag` will only ever have a hashtag value that was sent in the data sources schema, which is by definition a hashtag prefix.